src/Component/Rules/Rules.tsx

```diff
@@ -55,20 +55,17 @@ import { SortableItem } from '../SortableItem/SortableItem';
 import { RemovableItem } from '../RemovableItem/RemovableItem';
 import { useGeoStylerComposition } from '../../context/GeoStylerContext/GeoStylerContext';
 
-// default props
-interface RulesDefaultProps {
+export interface RulesComposableProps {
+  disableClassification?: boolean;
```

Comment:
IMHO all boolean properties should have the default value to be falsy. Its kind of a  The benefit is that using the component like this makes it possible to configure properties like this:

```jsx
<Rules disableClassification />
```

instead of writing this (double negation)

```jsx
<Rules enableClassification={false} />
```